Helm/opensearch-dashboards/templates/deployment.yaml

```diff
@@ -117,7 +117,7 @@ spec:
               key: cookie
             {{- end }}
 {{- if .Values.extraEnvs }}
-{{ toYaml .Values.extraEnvs | indent 10 }}
+{{ toYaml .Values.extraEnvs | indent 8 }}
```

Comment:
Can we also just indent this in the file? Changes look good, just might be worth doing some housekeeping and getting these indentations in the file done too. 

@TheAlgo Any reason why we didn't indent everything? This hurts folds in IDEs and just doesn't look great. It's a personal preference though.

Reply:
@DandyDeveloper I was indenting while testing following a TDD model. I prefer this model instead of changing all things at once.